### ISSUE TITLE
Possible to use contextily backgrounds? 

### ISSUE THREAD

**Issue by cwerner** [lukelbd/proplot#161]
### Description

Hi. I try to include some tile backgrounds using contextily, however I fail to do so. Is this supported?

### Steps to reproduce

```python
import proplot as plot
import contextily as ctx
 
f, axs = plot.subplots(ncols=2, nrows=2, proj={k:'merc' for k in range(1,5)})

for i in range(4):
    axs[i].set_extent([100, 102, 30, 32])
    ctx.add_basemap(axs[i], zoom=10, transform=ccrs.Mercator(), url=ctx.providers.Stamen.TonerLite)
```

**Expected behavior**: 

Get the tile STAMEN TonerLite as background

**Actual behaviour**:
Error:
```
RuntimeError: Invalid plotting method 'imshow' for map projection axes.
```

### Proplot version

v0.5

**Comment by lukelbd** [comment]
This was fixed by PR #157 (see also #147).

**Comment by cwerner** [comment]
Great!
